include/mbedtls/build_info.h

```diff
@@ -101,6 +101,9 @@
 #define inline __inline
 #endif
 
+#undef MBEDTLS_CONFIG_FILES_READ
+#undef MBEDTLS_CONFIG_IS_FINALIZED
```

Comment:
The undefs are to avoid unexpected behavior if users try to work around the config file inclusion. Erroring out also works, I don't think it makes a practical difference, so ok.